index.html

```diff
@@ -0,0 +1,230 @@
+                    Hey There.
+                    I'm Okari.
+                </h1>
+                <h2 class="job-title">
+                    I'm a software developer
+                </h2>
+                <p class="job-description">
+                    I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+                </p>
+                <ul class="links">
+                    <li class="socials">
+                        <a class="icon-links" href="https://github.com">
+                            <img src="images/Icon -GitHub.svg" alt="github-icon" class="image-icons">
+                        </a>
+                    </li>
+                    <li class="socials">
+                        <a class="icon-links" href="https://linkedin.com">
+                            <img src="images/Icon - Linkedin.svg" alt="linkedin-icon" class="image-icons">
+                        </a>
+                    </li>
+                    <li class="socials">
+                        <a class="icon-links" href="https://angelist.com">
+                            <img src="images/Icon - Angelist.svg" alt="angellist-icon" class="image-icons">
+                        </a>
+                    </li>
+                    <li class="socials">
+                        <a class="icon-links" href="https://twitter.com">
+                            <img src="images/Icon - Twitter.svg" alt="twitter-icon" class="image-icons">
+                        </a>
+                    </li>
+                    <li class="socials">
+                        <a class="icon-links" href="https://medium.com">
+                            <img src="images/Icon - Medium.svg" alt="medium-icon" class="image-icons">
+                        </a>
+                    </li>
+                </ul>
+            </article>
+        </section>
+        <!--Work section and about section-->
+        <section class="section-2">
+            <article class="recent-article">
+                <h2 class="recent-text">My Recent Works</h2>
+                <div class="recent-break"></div>
+                <img src="images/multipost.svg" alt="proj-image" class="recent-img">
+                <h3 class="recent-text-2">Multi-Post Stories</h3>
+                <p class="recent-text-3">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <div class="recent-btns">
+                    <button class="recent-css-btn">CSS</button>
+                    <button class="recent-html-btn">HTML</button>
+                    <button class="recent-bootstrap-btn">Bootstrap</button>
+                    <button class="recent-ruby-btn">Ruby</button>
+                </div>
```

Comment:
- [ ] According to the project requirement, the `<div>` tag should be replaced with `<ul>` tag and the `<button>` be replaced with `<li>` tags. They are supposed to be list items. Below is the screenshot of the requirement.
![image](https://github.com/2004-okari/MY-PORTFOLIO/assets/43528594/7cf65549-f65d-4c62-8d1a-26c09d5cd505)

Please ensure that you follow the requirement accordingly. :slightly_smiling_face: